ionic2/src/pages/home/home.scss

```diff
@@ -1,3 +1,132 @@
 page-home {
+    //@extend .deflash-blue;
```

Comment:
Can we do this as a move rather than an add? (We can delete this file from the old `sass` dir.) Would help to be able to track where things are coming from.